.changes/v3.13.0/1244-bug-fixes.md

```diff
@@ -0,0 +1 @@
+* Fix Issue [#1242](https://github.com/vmware/terraform-provider-vcd/issues/1242): panic when edge gateway IP count returns empty [GH-1244]
```

Comment:
If we release this as a bugfix, we can move it already to v3.12.1

Reply:
changed